litmus-portal/frontend/src/pages/MonitoringDashboardPage/index.tsx

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
-import { useQuery } from '@apollo/client';
```

Comment:
This index file is very big. Please try to break into smaller components

Reply:
The state is tied to this file as its complex, all components are inter dependent other than chaos table. If we break the logic it too many extra rendering would be there. Will take a look if static ones can be isolated.

Reply:
Try it otherwise this file will become hard to edit in future

Reply:
Please try to break down this file